Dockerfile

```diff
@@ -56,4 +56,6 @@ RUN yarn workspace @zooniverse/classifier build
 
 RUN yarn workspace @zooniverse/fe-content-pages build
 
+RUN echo $COMMIT_ID > /usr/src/packages/app-project/public/commit_id.txt
```

Comment:
Minor - I'd probably put this after the following `RUN yarn workspace @zooniverse/fe-project build` cmd to avoid any change in the COMMIT_ID invalidating the image cache for the fe-project build step - save the rebuilding times for no changes.

Reply:
I think the static assets might need to be in place before `.next` is built, but I’m not sure.